src/Compilers/Core/Portable/Text/TextChangeEventArgs.cs

```diff
@@ -21,7 +21,7 @@ public class TextChangeEventArgs : EventArgs
         /// <param name="changes">A non-empty set of ranges for the change.</param>
```

Comment:
Docs would need changing. Any idea if anything in the compiler relied on this?

Reply:
so looked around. so it is only used only for
http://source.roslyn.io/#Microsoft.CodeAnalysis/Text/SourceTextContainer.cs,21

and in compiler, text never changes, so there is no one consuming the event.